components/collapsible-panel/collapsible-panel.js

```diff
@@ -0,0 +1,344 @@
+import '../colors/colors.js';
+import '../icons/icon.js';
+import { css, html, LitElement } from 'lit';
+import { heading1Styles, heading2Styles, heading3Styles, heading4Styles } from '../typography/styles.js';
+import { hide, show } from '../../directives/animate/animate.js';
+import { classMap } from 'lit/directives/class-map.js';
+import { RtlMixin } from '../../mixins/rtl-mixin.js';
+
+// TODO: should we export/share this? I stole this from calendar
+const keyCodes = {
+	ENTER: 13,
+	SPACE: 32,
+};
+
+/**
+ * A container with a title that can be expanded/collapsed to show/hide content.
+ * @slot header - Slot for header content, such as course image (no actionable elements)
+ * @slot summary - Slot for the summary of the expanded content. Only accepts `d2l-collapsible-panel-summary-item`
+ * @slot default - Slot for the expanded content
+ * @slot actions - Slot for buttons and dropdown openers to be placed in top right corner of header
+ * @fires d2l-collapsible-panel-expand - Dispatched when the panel is expanded
+ * @fires d2l-collapsible-panel-collapse - Dispatched when the panel is collapsed
+ */
+class CollapsiblePanel extends RtlMixin(LitElement) {
+
+	static get properties() {
+		return {
+			/**
+			 * REQUIRED: The title of the panel
+			 * @type {string}
+			 */
+			title: { type: String },
+			/**
+			 * The semantic heading level (h1-h5)
+			 * @type {'1'|'2'|'3'|'4'|'5'}
```

Comment:
Are these intended to go to `5`? In the header styling method I see code only checking for `1-4`.

Reply:
Ah, for some reason I thought core supported 1-5, but looks like we only support 1-4.